website/docs/r/cloudfront_distribution.html.markdown

```diff
@@ -273,7 +273,8 @@ of several sub-resources - these resources are laid out below.
     response to requests using the specified HTTP methods.
 
 * `cache_policy_id` (Optional) - The unique identifier of the cache policy that
-    is attached to the cache behavior.
+    is attached to the cache behavior. Either [`cache_policy_id` and optional `origin_request_policy_id`] or
```

Comment:
To provide some clarity, the CloudFront Distributions are required to have a DefaultCacheBehavoir, and optionally can have CacheBehaviors specified (via the schema attribute `ordered_cache_behavoir`).

When configuring the DefaultCacheBehaoir the user must include either a CachePolicyId or ForwardedValues, and is separate from speciying OriginRequestPolicyId.

```suggestion
    is attached to the cache behavior. If configuring the `default_cache_behavior` either `cache_policy_id` or `forwarded_values` must be set.
```